packages/react/use-overlay/src/tests/exit.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { act, screen, waitFor } from '@testing-library/react';
+import React, { useEffect } from 'react';
+import { useOverlay } from '../useOverlay';
+import { renderWithContext } from './utils';
+
+const closeDuration = 1000;
+
+function OverlayCallerComponent() {
+  const overlay = useOverlay();
+
+  useEffect(() => {
+    overlay.open(() => {
+      return <div>toss</div>;
+    });
+
+    const timeoutID = setTimeout(() => {
+      overlay.exit();
+    }, closeDuration);
+
+    return () => clearTimeout(timeoutID);
+  }, [overlay]);
+
+  return null;
```

Comment:
What about adding buttons like `open`, and `exit` here, which opens the overlay and unmounts it, and writing the code as below?

```tsx
const openButton = screen.getByText('open');
const exitButton = screen.getByText('exit');

await userEvent.click(openButton);

expect(screen.getByText('toss')).toBeInTheDocument();

await userEvent.click(closeButton);

expect(screen.queryByText('toss')).not.toBeInTheDocument();

Reply:
This test code will not require `jest.advanceTimersByTime()` so I think it's much simpler.

Reply:
Thx I made it!